scripts/AsmJsonImportTest.sh

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
```suggestion
set -eu
```

Or even this (if it works):
```suggestion
set -euo pipefail
```